models/iris_depth_camera/model.config

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<model>
+  <name>3DR Iris with depth camera</name>
+  <version>1.0</version>
+  <sdf version='1.4'>iris_depth_camera.sdf</sdf>
```

Comment:
You state in the `iris_depth_camera.sdf` that the SDF version is 1.5 and here that is 1.4. Please change it to 1.5 so to keep it consistent.